src/Components/ExternalResult/ExternalResultUpload.tsx

```diff
@@ -111,7 +112,13 @@ export default function ExternalResultUpload() {
                 />
                 <a
                   className="focus:ring-blue mx-auto mt-4 max-w-xs items-center rounded-md border border-primary-500 bg-white px-3 py-2 text-sm font-medium leading-4 text-primary-700 transition duration-150 ease-in-out hover:text-primary-500 hover:shadow focus:border-primary-300 focus:outline-none active:bg-gray-50 active:text-primary-800"
-                  href={sample_format_external_result_import}
+                  href="#"
+                  onClick={() =>
+                    downloadFile(
+                      sample_format_external_result_import,
+                      "sample_format"
+                    )
+                  }
```

Comment:
instead of custom download function, why not try download attribute, something like
```suggestion
                  href={sample_format_external_result_import}
                  target="_blank"
                  download
```

Reply:
![image](https://github.com/coronasafe/care_fe/assets/101407963/671d74dd-c3fb-40b9-b8ec-87ef500b9642)

I believe that adding a download attribute to the link allows for direct downloading, but this method may not be preferable. On hover, we can disclose the path, though this approach is also not ideal.

Should I proceed with making these changes, or would you prefer that I leave it as it is?

`http://localhost:4000/External-Results-Template.csv`

Reply:
Is disclosing the path an issue/concern?

Reply:
Why is it not ideal? The lesser the javascript, the better (if it does the same thing and has no compromises).

We are practically doing the same thing with the `downloadFile` utility too anyways right?

<img width="614" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/e9df643d-adad-4c8e-bebc-12dd1ad2a279">